src/cljs/rems/administration/catalogue_items.cljs

```diff
@@ -116,6 +120,7 @@
      [:label.form-check-label {:for "display-archived"}
       (text :t.administration/display-archived)]]))
 
+
```

Comment:
Extra newline?

Reply:
To separate the archiving related functions from the rest.